dist/angular-ui-tree.js

```diff
@@ -796,6 +796,7 @@
             var dragStarted = false;
             var dragTimer;
             var body = document.body,
```

Comment:
Sometimes body.getAttribute is used in the code; I didn't clean up all to use consistently angular elements only.